### PR TITLE
Docker ce upgarde to mew board - note addition to the doc and screens…

### PR DESCRIPTION
**Pull request by sumoanema**
…hot update

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/containers-orchestration/docker-community-edition.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/integrations/containers-orchestration/docker.png')} alt="icon" width="150"/>
 
+:::note
+The Docker App will be fully functional working with Docker setup using cgroup v1. For more details on cgroup click [here](https://docs.docker.com/config/containers/runmetrics/#control-groups).
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/docker-community-edition.md:13]:
```suggestion
The Docker app will be fully functional with Docker setup using cgroup v1. [Learn more]https://docs.docker.com/config/containers/runmetrics/#control-groups) about cgroup.
```

File: docs/integrations/containers-orchestration/docker-community-edition.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+:::
+
 The Sumo Logic App for Docker Community edition (CE) provides operational insight into your Docker containers. The App includes Dashboards that allow you to view your container performance statistics for CPU, memory, and the network. It also provides visibility into container events such as start, stop, and other important commands.
```
Comment (JV0812) [inline docs/integrations/containers-orchestration/docker-community-edition.md:16]:
```suggestion
The Sumo Logic app for Docker Community edition (CE) provides operational insight into your Docker containers. This app includes dashboards that allow you to view your container performance statistics for CPU, memory, and the network. It also provides visibility into container events such as start, stop, and other important commands.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor formating

**Comment by sumoanema** [comment]
Done with implementing the suggested changes. Can you please review @JV0812

**Review by JV0812 — APPROVED** [review]

File: docs/integrations/containers-orchestration/docker-community-edition.md [hunk 3]
```diff
@@ -9,7 +9,11 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/integrations/containers-orchestration/docker.png')} alt="icon" width="150"/>
 
-The Sumo Logic App for Docker Community edition (CE) provides operational insight into your Docker containers. The App includes Dashboards that allow you to view your container performance statistics for CPU, memory, and the network. It also provides visibility into container events such as start, stop, and other important commands.
+:::note
+The Docker app will be fully functional with Docker setup using [cgroup v1](https://docs.docker.com/config/containers/runmetrics/#control-groups).
+:::
+
+The Sumo Logic app for Docker Community edition (CE) provides operational insight into your Docker containers. This app includes dashboards that allow you to view your container performance statistics for CPU, memory, and the network. It also provides visibility into container events such as start, stop, and other important commands.
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/docker-community-edition.md:16]:
```suggestion
The Sumo Logic app for Docker Community Edition (CE) provides operational insight into your Docker containers. This app includes dashboards that allow you to view your container performance statistics for CPU, memory, and the network. It also provides visibility into container events such as start, stop, and other important commands.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**